src/__tests__/lib/igloo-core-integration.test.ts

```
@@ -0,0 +1,504 @@
+        group: 'test-group',
+        share: 'test-share',
+        relays: ['wss://relay.test.com']
+      });
+      
+      expect(result.state.isConnected).toBe(true);
+      expect(result.state.isReady).toBe(false);
+    });
+  });
+
+  describe('Event Handling', () => {
+    let eventHandlers: Map<string, Function[]>;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)